urbs.py

```diff
@@ -1894,18 +1895,27 @@ def report(instance, filename, commodities=None, sites=None):
             for sit in sites:
                 created, consumed, stored, imported, exported, derivative = get_timeseries(
                     instance, co, sit)
+                
+                # temporarily drop out the timeseries demand and delta, for a correct overproduction calculation
```

Comment:
These two columns always exists at the moment. At first I was trying to just add the columns needed (Feed-in and Demand/Shifted Demand). But Feed-in does not always exist, and I think this one scales better than having a try and error environment around it.
I am open to better suggestions!